plugins/services/src/js/reducers/serviceForm/PortMappings.js

```diff
@@ -119,19 +119,28 @@ module.exports = {
         });
       }
 
-      const vip = findNestedPropertyInObject(item, `labels.VIP_${index}`);
+      const vip = Object.entries(item.labels || {}).find(([key, _]) =>
```

Comment:
I see this `.find(([key, _]) => key.match(/^(vip|VIP)/)` a third time and it is a bit cryptic to me, which is normal with regexes I guess. Can we move it into a util, I think there already is one for labels

Reply:
any reason not to just use a case insensitive search? `key.match(/^vip/i)`

Reply:
Yeah, I read somewhere that VIP and vip is okay, Vip or vIP not

Reply:
makes sense, thanks

Reply:
@DanielMSchmidt ✅